### PR TITLE
Add types to HandDraw

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: js/src/HandDraw.ts [hunk 1]
```diff
@@ -28,163 +27,155 @@ export class HandDraw extends Interaction {
...
 
-    // Update line index
-    this.update_line_index();
-    this.listenTo(this.model, 'change:line_index', this.update_line_index);
-    this.model.on_some_change(['min_x', 'max_x'], this.set_limits, this);
+    this.model.on_some_change(['min_x', 'max_x'], this.setLimits, this);
   }
 
-  set_lines_view() {
+  private async setLinesView(): Promise<void> {
```
Comment (marimeireles) [inline js/src/HandDraw.ts:40]: methods names should be snake case?

Comment (marimeireles) [reply js/src/HandDraw.ts:40]: or is it overriding something else?

**Review by marimeireles — COMMENTED** [review]
Other than this one question, looking good! :)

File: js/src/HandDraw.ts (hunk 1 above)
Comment (martinRenou) [reply js/src/HandDraw.ts:40]: Nope, I think we decided to follow the JupyterLab conventions, so no more snake_case :)

Comment (marimeireles) [reply js/src/HandDraw.ts:40]: oh!
and pretty is not complaining? so ok :)

Comment (martinRenou) [reply js/src/HandDraw.ts:40]: Some of them are still snake_case because I am waiting for us to work on super classes like `Mark` before renaming them

**Pull request merged**